S/SEAL_jll/build_tarballs.jl

```diff
@@ -0,0 +1,76 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "SEAL_jll"
```

Comment:
This should be
```suggestion
name = "SEAL"
```
The `_jll` is automatically appended in the JLL package.  Note that you must also rename the directory to exactly match this name

Reply:
I am very sorry about that, this clearly was an oversight... Fixed in 2a36fd1.